op-e2e/actions/proofs/operator_fee_test.go

```diff
@@ -0,0 +1,147 @@
+package proofs
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	actionsHelpers "github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/proofs/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/bindings"
+	"github.com/ethereum-optimism/optimism/op-program/client/claim"
+	"github.com/ethereum-optimism/optimism/op-service/predeploys"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+const OperatorFeeScalar = uint32(20000)
+const OperatorFeeConstant = uint64(500)
```

Comment:
I would prefer to indicate in the variable names that these are for tests, since they are global variables, like
```suggestion
const testOperatorFeeScalar = uint32(20000)
const testOperatorFeeConstant = uint64(500)
```
It's also unusual to use names in upper case, as tests usually don't export variables.

And even better would be to move the consts into the test `Test_Operator_Fee_Constistency` if they are not used anywhere else.

Reply:
Done